client/src/app/site/pages/meetings/pages/participants/modules/groups/components/group-list/group-list.component.ts

```diff
@@ -252,4 +252,19 @@ export class GroupListComponent extends BaseMeetingComponent implements OnInit {
             this.updatedGroupIds.clear();
         };
     }
+
+    /**
+     * Function to open a prompt dialog,
+     * so the user will be warned if he has made changes and not saved them.
+     *
+     * @returns The result from the prompt dialog.
+     */
+    public async canDeactivate(): Promise<boolean> {
+        if (this.hasChanges) {
+            const title = this.translate.instant(`Do you really want to exit this page?`);
+            const content = this.translate.instant(`You made changes.`);
+            return await this.promptService.open(title, content);
```

Comment:
There are multiple places where such a prompt is opened (just search for "you made changes"), they should be unified with the slightly differently worded but similar function `discardChangesConfirmation()` in the `PromptService`.